src/platform_impl/macos/mod.rs

```diff
@@ -98,6 +99,12 @@ impl Window {
     let () = unsafe { msg_send![*self.delegate, clearIsCheckingZoomedIn] };
     f
   }
+
+  #[inline]
+  pub fn is_minimized(&self) -> bool {
+    let is_minimized: BOOL = unsafe { msg_send![*self.ns_window, isMiniaturized] };
+    is_minimized == YES
+  }
```

Comment:
could you move this function and the one above it (is_maximized) to `window.rs`? I don't know why it was defined there.

Reply:
fixed on e3a0a8d